civihr_employee_portal/views/includes/views_between_operation_dates_filter_handler.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @file
+ * A filter that allows the between operator to compare dates in 'Y-m-d' format
+ * rather than the default timestamp comparison since absence dates are stored
+ * in 'Y-m-d' format in the db.
+ */
+
+class views_between_operation_dates_filter_handler extends views_handler_filter_date {
```

Comment:
It's annoying that Drupal doesn't have default handlers for this kind of thing :-(